activemodel/lib/active_model/dirty.rb

```diff
@@ -174,12 +177,10 @@ def changed_attributes
     end
 
     # Handles <tt>*_changed?</tt> for +method_missing+.
-    def attribute_changed?(attr, options = nil) #:nodoc:
+    def attribute_changed?(attr, from: OPTION_NOT_GIVEN, to: OPTION_NOT_GIVEN) # :nodoc:
       result = changes_include?(attr)
-      if options
-        result &&= options[:to] == __send__(attr) if options.key?(:to)
-        result &&= options[:from] == changed_attributes[attr] if options.key?(:from)
-      end
+      result &&= to == __send__(attr) unless to == OPTION_NOT_GIVEN
```

Comment:
Do you think this is a good opportunity to break this into multiple lines? It's very difficult to parse mentally in this form.